wandb/sdk/artifacts/utils.py

```diff
@@ -0,0 +1,5 @@
+REGISTRY_PREFIX = "wandb-registry-"
```

Comment:
nit: for extra safety in lint checks, you can type hint constants with `Final` (since python 3.8, so extra imports below are for 3.7 compatibility)

this way type checks should fail on the off-chance `REGISTRY_PREFIX` is ever inadvertently mutated

```suggestion
from __future__ import annotations

from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from typing_extensions import Final

REGISTRY_PREFIX: Final[str] = "wandb-registry-"
```